testing/generate.js

```diff
@@ -4,8 +4,8 @@ const path = require('path');
 const slugify = require('slugify');
 const yaml = require('js-yaml');
 
-const HOSTNAME = 'api';
-// const HOSTNAME = 'localhost';
+// const HOSTNAME = 'api';
```

Comment:
hm ideally we don't leave commented out code in commits. Is there a good reason to have this, and would a comemnt explaining when you'd want to change it to api be better?

Reply:
Kyle originally had this as well. Reason to have this because we would flip between `localhost` and `api` based on whether the Playwright tests are being executed inside of a Docker container or not. In the v1 version, they were executed in a Docker container so we used `api` and in v2, they're executed in the Github Actions VM, so we use `localhost`. Will add a comment explaining that.